docs/learn/releases.rst

```diff
@@ -94,21 +103,55 @@ look like this:
         headers={'Authorization': 'Bearer {}'.format(SENTRY_API_TOKEN)},
     )
 
+For more information, you can check out our
+:doc:`API <../api/releases/post-organization-releases/>`
+or :ref:`CLI <sentry-cli-commit-integration>` docs.
+
+
+Resolving issues via commits
+----------------------------
+
 Once you are sending commits (either as ``commits`` or ``refs``), you
 can start including ``fixes <SHORT-ID>`` in your commit messages. Then,
 once we identify a commit as being included in a release, we'll
 automatically resolve that issue. You can find the short issue id at
 the top of the issue details page, next to the assignee dropdown.
 
-We'll also start suggesting assignees for issues based on changes
-to files in an issue's stack trace once we have commit data.
+For example, a commit message might look like this:
 
-For more information, you can check out our
-:doc:`API <../api/releases/post-organization-releases/>`
-or :ref:`CLI <sentry-cli-commit-integration>` docs.
+.. code-block::
+
+    Prevent empty queries on users
+
+    Fixes SENTRY-317
+
+
+When Sentry sees this commit, we'll automatically annotate the matching
+issue with a reference to the commit, and upon deploy, we'll mark the issue
```

Comment:
uhh i have to double check, but i think it is actually when release is created not when deploy is created if that matters